src/angular/4-adding-routing/adding-routing.md

```diff
@@ -106,32 +95,38 @@ This example shows one route redirecting to another:
 
 ## Setting Paths to Modules
 
-As our applications grow, it doesn’t make sense to load all the code at once. Thanks to <a href="https://angular.io/guide/lazy-loading-ngmodules" >lazy loading</a>, we can wait to render modules until a specific route requiring them is hit.
+As our applications grow, it doesn’t make sense to load all the code at once. Thanks to <a href="https://angular.io/guide/lazy-loading-ngmodules">lazy loading</a>, we can wait to render modules until a specific route requiring them is hit.
 
-@sourceref ./lazyload-route.html
-@codepen
-@highlight 120, only
+<!-- [//]: # @sourceref ./lazyload-route.html
+[//]: # @codepen
+[//]: # @highlight 120, only -->
+
+```typescript
+const routes: Routes = [
+  { path: 'about', component: AboutComponent },
+  { path: 'colophon',  redirectTo: '/about', pathMatch: 'full' },
+  {
+    path: 'products',
+    loadChildren: () => {
+      return import('./products/products.module').then(m => m.ProductsModule)
+    }
```

Comment:
@fabioemoutinho May I get your confirmation that this looks right please? I wanted to make the example more realistic and improve the formatting.

Reply:
usually we return immediately:
```loadChildren: () => import('./products/products.module').then(m => m.ProductsModule)```
https://angular.io/guide/lazy-loading-ngmodules

Reply:
Yeah, I saw that syntax but I think it’s more difficult to understand because there are multiple implicit returns. I’m going to keep it as is.

Reply:
my only concern is that users will hardly ever see the explicit return in these files (I've never seen it on a project) since the official docs and examples always use implicit returns, but it's fine, I guess.